packages/main/src/ColorPalette.js

```diff
@@ -50,6 +54,28 @@ const metadata = {
 			type: Boolean,
 		},
 
+		/**
+		 * Defines whether the user can choose the default color from a button.
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @private
+		 * @since 1.0.0-rc.16
+		 */
+		showDefaultColor: {
+			type: Boolean,
+		},
+
+		/**
+		 * Defines the default color of the color palette
+		 * <b>Note:</b> The default color should be a part of the ColorPalette colors</code>
+		 * @type {CSSColor}
+		 * @private
+		 * @since 1.0.0-rc.16
```

Comment:
add defaulvalue tag

Reply:
Do we need default value tag for private property? It will be empty string.